### DISCUSSION TITLE
diary/2024/12/28/2024-retrospective/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#161] [Comments]
# diary/2024/12/28/2024-retrospective/

2024년도 끝나가네요. 매년 하던 것처럼 올해도 회고를 작성해봅니다 과거 회고 글 모음 데이터 사이언티스트가 되기 위해 진행한 다양한 노력들 2017년 회고, 2018년 계획 2018년 회고, 2019년 다짐 Gap Year 및 쏘카 이직 이야기 2019년 상반기 회고 및 글또 3기 시작 초보 개발 팀장의 1년 회고 - 좋은 팀장이 되기 위한...

https://zzsza.github.io/diary/2024/12/28/2024-retrospective/

**Comment by leojuun** [comment]
개발자는 아니지만 어쩌다 들어오게 된 블로그를 즐겨찾기에 저장해 놓고 가끔 들어오는데 꾸준히 올리시는 회고록을 보고 저를 돌아보곤 합니다. 잘 읽었습니다. 저도 목표를 세우고 가슴이 두근거리는 삶을 만들어 봐야겠습니다.
